tests/accessor/host_accessor_constructors_fp64.cpp

```diff
@@ -24,10 +24,17 @@ DISABLED_FOR_TEST_CASE(hipSYCL, ComputeCpp, DPCPP)
 ("sycl::host_accessor constructors. fp64 type", "[accessor]")({
   using namespace host_accessor_constructors;
   auto queue = sycl_cts::util::get_cts_object::queue();
-#if !SYCL_CTS_ENABLE_FULL_CONFORMANCE
-  run_host_constructors_test<double>{}("double");
-#else
+  if (!queue.get_device().has(sycl::aspect::fp64)) {
+    WARN(
+        "Device does not support double precision floating point operations. "
+        "Skipping the test case.");
+    return;
+  }
+
+#if SYCL_CTS_ENABLE_FULL_CONFORMANCE
   for_type_vectors_marray<run_host_constructors_test, double>("double");
+#else
+  run_host_constructors_test<double>{}("double");
 #endif  // SYCL_CTS_ENABLE_FULL_CONFORMANCE
-})
+});
```

Comment:
To be honest, I'm not sure if `;` is needed after `DISABLED_FOR_TEST_CASE` macro.

Reply:
We have an example of using that has been provided by Philip Salzmann and this example contained the `;`
This example can be found [here](https://github.com/KhronosGroup/SYCL-CTS/blob/SYCL-2020/tests/common/disabled_for_test_case.h#L30)

Reply:
According to my understanding the macro is expanded into either a function definition or no-op. Does it sound right to you? Both of these bases shouldn't require `;` at the end.
It's minor issue as extra `;` doesn't break anything.

Reply:
> Does it sound right to you?

Yes, I think that I am agreed with you

> It's minor issue as extra ; doesn't break anything.

So, let's don't delete extra `;`